test/unit/Dependencies.js

```diff
@@ -72,7 +73,7 @@ test('it can utilize custom checks for a dependency', t => {
             check: postcss => {
                 called = true;
 
-                t.true(postcss().version.startsWith('8.1'));
+                t.true(semver.satisfies(postcss().version, '^8.1'));
```

Comment:
You know… if I'd've known we already had the semver package installed I would've used it. Definitely a much better option.

👍

Reply:
I know. So adding semver looks very good.